tests/test_arcaflow_plugin_stressng.py

```diff
@@ -62,13 +69,22 @@ def test_functional_cpu(self):
 
         self.assertEqual(yaml.safe_load(result), reference)
         res = stressng_plugin.stressng_run(self.id(), stress)
+        print(res)
```

Comment:
Do we want to merge this with what I presume is a diagnostic print statement?

Ditto line 107, et al.

Reply:
IMO as part of the test code, this enables better visibility.